src/sap.m/src/sap/m/ListItemBase.js

```diff
@@ -49,6 +49,11 @@ sap.ui.define(['jquery.sap.global', './library', 'sap/ui/core/Control', 'sap/ui/
 			unread : {type : "boolean", group : "Misc", defaultValue : false},
 
 			/**
+			 * Whether the control should be enabled. If set to false, the item is rendered as disabled.
+			 */
+			enabled : {type : "boolean", group : "Appearance", defaultValue : true},
```

Comment:
@aborjinik in multi-select mode they would not be selectable (and disabled)  but in single-select mode they would be disabled. 
Usually the "visual" effect is that the item is disabled (grey) and not clickable. That's why I used "enabled" as the name of the property.

Anyway, that would not be a problem to change the name, if that's the issue